plugins/mongodb/mongodb.ecllib

```diff
@@ -26,3 +26,6 @@ EXPORT boolean supportsScript := true;
 EXPORT updateResultRecord := {INTEGER matched_count, INTEGER modified_count};
 EXPORT insertManyResultRecord := {INTEGER inserted_count};
 EXPORT deleteResultRecord := {INTEGER deleted_count};
+
+EXPORT regexType := {STRING pattern, STRING options};
+EXPORT timestampType := {UNSIGNED t, UNSIGNED i};
```

Comment:
Can we document these fields, or provide a URL to a MongoDB page describing their usage?